datasets/duorc/README.md

```diff
@@ -0,0 +1,161 @@
+
+### Data Fields
+
+- `plot_id`: a `string` feature containing the movie plot ID.
+- `plot`: a `string` feature containing the movie plot text.
+- `title`: a `string` feature containing the movie title.
+- `question_id`: a `string` feature containing the question ID.
+- `question`: a `string` feature containing the question text.
+- `answers`: a `list` of `string` features containing list of answers.
+- `no_answer`: a `bool` feature informing whether the question has no answer or not.
+
+
+### Data Splits
+
+The data is split into a training, dev and test set in such a way that the resulting sets contain 70%, 15%, and 15% of the total QA pairs and no QA pairs for any movie seen in train are included in the test set. The final split sizes are as follows:
+
+Name Train Dec Test
+SelfRC 60721 12961 12599
+ParaphraseRC 69524 15591 15857
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+Wikipedia and IMDb movie plots
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+For SelfRC, the annotators were allowed to mark an answer span in the plot or synthesize their own answers after reading Wikipedia movie plots. 
+For ParaphraseRC, questions from the Wikipedia movie plots from SelfRC were used and the annotators were asked to answer based on IMDb movie plots.
+
+#### Who are the annotators?
+
+Amazon Mechanical Turk Workers
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+The dataset was intially created by Amrita Saha, Rahul Aralikatte, Mitesh M. Khapra, and Karthik Sankaranarayanan in a collaborated work between IIT Madras and IBM Research.
+
+### Licensing Information
+
+[MIT License](https://github.com/duorc/duorc/blob/master/LICENSE)
+
+### Citation Information
+
+@inproceedings{DuoRC,
+author = { Amrita Saha and Rahul Aralikatte and Mitesh M. Khapra and Karthik Sankaranarayanan},
+title = {{DuoRC: Towards Complex Language Understanding with Paraphrased Reading Comprehension}},
+booktitle = {Meeting of the Association for Computational Linguistics (ACL)},
+year = {2018}
+}
```

Comment:
````suggestion
```
@inproceedings{DuoRC,
author = { Amrita Saha and Rahul Aralikatte and Mitesh M. Khapra and Karthik Sankaranarayanan},
title = {{DuoRC: Towards Complex Language Understanding with Paraphrased Reading Comprehension}},
booktitle = {Meeting of the Association for Computational Linguistics (ACL)},
year = {2018}
}
```
````